xcape/core.py

```diff
@@ -22,15 +22,23 @@ def _reshape_inputs(*args):
 
     args_al2d = [np.atleast_2d(a) for a in args]
     original_shape = args_al2d[0].shape
-    new_shape = (_prod(original_shape[:-1]),) + (original_shape[-1],)
+    # for now CAPE works with shape [lev , prod(lat*lon*time)]
+    # lev is the first dimension
+    #new_shape = (_prod(original_shape[:-1]),) + (original_shape[-1],)
+    new_shape = (original_shape[0],) + (_prod(original_shape[1:]),) 
```

Comment:
I now have the input data as:
(level, dim1, dim2, dim3, ...) 
levels are the first dimension. 
And ALLCAPELOOP wants the data as:
(lev, #gridpoints)

I modified the code accordingly

Reply:
This makes sense.

With git, you don't have to comment out old code. It automatically keeps track of diffs, as you can see. So please delete the commented out code.